addons/website_sale_digital/views/website_sale_digital.xml

```diff
@@ -1,20 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <template id="assets_backend" inherit_id="web.assets_backend">
+        <xpath expr="." position="inside">
+            <script type="text/javascript" src="/website_sale_digital/static/src/js/sidebar.js"></script>
+        </xpath>
+    </template>
+
+    <template id="digital_content_btn">
```

Comment:
Please use meaningful and correctly prefixed names. It eases finding / searching / understanding.